driver_go1.10.go

```diff
@@ -19,10 +21,16 @@ var (
 // WrapConnector allows wrapping a database driver.Connector which eliminates
 // the need to register ocsql as an available driver.Driver.
 func WrapConnector(dc driver.Connector, options ...TraceOption) driver.Connector {
-	opts := TraceOptions{}
+	opts := TraceOptions{
```

Comment:
Let's default to empty here too, just like with the `Wrap` function.

Reply:
Done, please check latest commit.